tests/slow/src/test/scala/tests/feature/Scala3CodeActionLspSuite.scala

```diff
@@ -131,10 +132,9 @@ class Scala3CodeActionLspSuite
     """|object Main {
        |  def method2(i: Int) = ???
        |  def main = {
-       |    def inner(i : Int) = {
```

Comment:
I think this one should stay the same since main has braces, we also want inner to have braces.

Reply:
hmm… so by changing the code the way you suggested it above, the test would work?
I would try...

Reply:
It should work, I compared the changes in the methods.